### PR TITLE
Modify files for Windows unattended install

### PR DESCRIPTION
**Pull request by pablo-herranz**
Modified the .xml files to add a non-expiring password command. Added the bash script used for creating the unattend ISOs

### CODE REVIEW

File: data/wsl/create_unattend_iso.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+ #!/bin/bash
```
Comment (ricardobranco777) [inline data/wsl/create_unattend_iso.sh:1]: Remove leading space:

```suggestion
#!/bin/bash
```

File: data/wsl/create_unattend_iso.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+if [ -z $1 ]; then
```
Comment (ricardobranco777) [inline data/wsl/create_unattend_iso.sh:3]:
```suggestion
if [ -z "$1" ]; then
```

**Review by ricardobranco777 — APPROVED** [review]
Small suggestions. LGTM

File: data/wsl/create_unattend_iso.sh (hunk 2 above)
Comment (pablo-herranz) [reply data/wsl/create_unattend_iso.sh]: Done!

File: data/wsl/create_unattend_iso.sh (hunk 1 above)
Comment (pablo-herranz) [reply data/wsl/create_unattend_iso.sh]: Done!

**Comment by clanig** [comment]
Thanks for this PR!
Could you please run: `shellcheck --shell=bash data/wsl/create_unattend_iso.sh`?
There are more suggestions and it would be great if you could resolve them.

**Comment by pablo-herranz** [comment]
> shellcheck --shell=bash data/wsl/create_unattend_iso.sh

Nice suggestion @clanig!! I've run it and modified the file :)

**Review by ricardobranco777 — APPROVED** [review]

**Pull request merged**